pennylane_lightning_kokkos/src/tests/Test_Backend_info.cpp

```diff
@@ -0,0 +1,135 @@
+#include "Bindings.hpp"
+#include "Error.hpp"
+#include "StateVectorKokkos.hpp"
+#include "TestHelpers.hpp"
+#include <algorithm>
+#include <cassert>
+#include <catch2/catch.hpp>
+#include <cctype>
+
+using namespace Pennylane;
+namespace {} // namespace
+
+TEMPLATE_TEST_CASE("Bindings::getConfig", "[Backend Info]", float) {
+
+    SECTION("Check string split") {
+        const std::string str = "Check string split!";
+        const auto str_list = string_split(str, " ");
+        PL_ASSERT(str_list[0] == "Check");
+        PL_ASSERT(str_list[1] == "string");
+        PL_ASSERT(str_list[2] == "split!")
```

Comment:
PL_ASSERT is a runtime macro for the library, that throws when the predicate fails.

Catch2 tests should use `CHECK` and `REQUIRES` to validate behaviour for the test-suite, otherwise nothing gets reported or tracked. See https://github.com/PennyLaneAI/pennylane-lightning-kokkos/blob/main/pennylane_lightning_kokkos/src/tests/Test_StateVectorKokkos_NonParam.cpp for example on how this is used.

The same follows for all the other test-cases.

Reply:
Thanks @mlxd for this!!!